src/Neo.VM/Types/Map.cs

```diff
@@ -29,7 +28,7 @@ public class Map : CompoundType, IReadOnlyDictionary<PrimitiveType, StackItem>
         /// </summary>
         public const int MaxKeySize = 64;
 
-        private readonly OrderedDictionary<PrimitiveType, StackItem> dictionary = new();
+        private readonly Collections.OrderedDictionary<PrimitiveType, StackItem> dictionary = new();
```

Comment:
Where is `Collections` pointing to; `system` or `vm`?

Reply:
VM